### ISSUE TITLE
Subtitles gets hidden behind the control bar in Chrome, Firefox & Edge

### ISSUE THREAD

**Issue by martinstender** [videojs/video.js#6207]
## Description
As far as I understand, subtitles should move up and out of the way of the control bar, when the control bar becomes visible (on hover, usually).
But in Chrome and Firefox (both macOs and Win10) and Edge in Win10, only the subsequent subtitles are moved up - so the subtitles that are already visible when the control bar becomes active, get covered up by the control bar.

## Steps to reproduce
In Chrome, Firefox or Edge, go to this codepen, press play and move the cursor away from the player, to make the control bar go away.
[https://codepen.io/martinstender/pen/mdbwwGo?editors=1010](https://codepen.io/martinstender/pen/mdbwwGo?editors=1010)
After approximately 9 seconds (when Elton sits down at the piano at home 🙂 ), hover over the player again and you should see the subtitles get covered up by the control bar.

### Expected
Again, as far as I understand, subtitles should move out of the way as the control bar becomes visible, but that only happens in Safari.

### Actual
Subtitles get hidden behind the control bar.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
v7.0
#### browsers
Chrome, FireFox, but not Safari
#### OSes
macOs and Windows 10
### plugins
None

**Comment by gkatsev** [comment]
Yeah, I can confirm it doesn't work properly. The captions container (text-track-display) does continue getting the `bottom: 3em;` css applied properly, so, something must've been done recently that made that not matter.

(Also, if you update to 7.6.4, you wouldn't need that weird `player.vhs = null` workaround, as we fixed the stuttering issue).

**Comment by gkatsev** [comment]
Trying it out, looks like this never worked. I went all the way back to 6.0 (which was released april 2017). https://codepen.io/gkatsev/pen/xxKLELa?editors=1010
Definitely should be fixed but good to know that it isn't actually a regression.

**Comment by martinstender** [comment]
Thanks for looking into it, @gkatsev.
And I'll upgrade to 7.6.4 right now :-)

**Comment by tsi** [comment]
We also noticed this issue, created https://github.com/videojs/video.js/pull/9023 in an attempt to fix.
